app/routes/posts/show.js

```diff
@@ -4,6 +4,11 @@ export default Ember.Route.extend({
   model(params) {
     return this.get('store').find('post', params.post_id);
   },
+  emptyTextArea: Ember.on('activate', function(){
+    setTimeout(function(){
+      $(".ember-text-area").val("").focus();
+    },10)
+  }),
```

Comment:
what are you trying to accomplish here? Where did you put class 'ember-text-area'?